packages/xstate-store/src/types.ts

```diff
@@ -11,6 +11,9 @@ export type EnqueueObject<TEmittedEvent extends EventObject> = {
     [E in TEmittedEvent as E['type']]: (payload: Omit<E, 'type'>) => void;
   };
   effect: (fn: () => void) => void;
+  spawn: <T extends StoreConfig<any, any, any>>(
+    config: T
+  ) => Store<any, any, any>;
```

Comment:
I'd remove this method from this PR. From what I understand, this is unrelated to the atoms functionality and there are no new tests for this here either.